UnderlyingOptionsPS/FTPS3-Compare.ps1

```diff
@@ -5,12 +5,6 @@
 #AWS Tools for Powershell
 #AWS CLI
 
-#Set permissions and import necessary libraries
-Set-ExecutionPolicy -Scope CurrentUser RemoteSigned  
```

Comment:
Can't set the ExecutionPolicy within a script - if it isn't already RemoteSigned, you'll never make it here